app/src/main/java/is/genki/bigredapp/android/GetRequest.java

```diff
@@ -80,4 +82,14 @@ private String downloadUrl(String myUrl) throws IOException {
         }
     }
 
+    public static boolean isConnected(Activity activity) {
```

Comment:
I factored this out here, but I haven't changed the original method to use this one yet. (I wish Java had C#'s extension methods.)